answerfile.py

```diff
@@ -220,6 +220,11 @@ def parseCommon(self):
             if bl not in ['' , 'grub2']:
                 raise AnswerfileException("Unsupported bootloader '%s'" % bl)
 
+        results['target-platform'] = ''
```

Comment:
I'm assuming this is intentionally left undocumented?

Reply:
Yes!